src/ui/app/main/components/LayoutEditor/LayoutCanvas.module.scss

```
@@ -176,57 +178,57 @@ $handle-edge: 8px;
 }
 
-// Resting-state corner indicator: a small L-shaped hook, the single
-// standard shared with the overlay's F9 drag mode (WidgetContainer.module.scss)
-// regardless of which resize directions a widget offers — aspect-locked/
-// autoHeight widgets simply omit the N/S or corner handles entirely, which
-// is expected, not an inconsistency.
-$corner-hook-size: 8px;
-$corner-hook-width: 2px;
-$edge-hover-thickness: 4px;
+$corner-hook-size: 16px;
+// One stroke weight for every grab mark: a corner and an edge bar are the same
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)